common/azure_auth.go

```diff
@@ -270,6 +277,19 @@ func (aa *AzureAuth) patRequest() tokenRequest {
 	}
 }
 
+func maybeExtendAuthzError(err error) error {
+	var ae APIError
+	fmtString := "Azure authorization error.  Does your SPN  have Contributor access to Databricks workspace. %v"
+	if errors.As(err, &ae) {
+		if ae.ErrorCode == "403" {
```

Comment:
isn't it `ae.StatusCode == 403` instead?

Reply:
yeah....